config/environments/development.rb

```diff
@@ -53,4 +53,7 @@
 
   # Default host for url generation
   routes.default_url_options[:host] = "localhost:#{ENV['PORT'] || 3000}"
+
+  # Enable curl for testing oauth flow
+  config.action_controller.allow_forgery_protection = false
```

Comment:
What error does this resolve? I'd expect our endpoints to work with cURL (or any non-browser system) in production.

Reply:
Leftover from one of my tests, sorry